src/js/popup/popup.js

```diff
@@ -1,4 +1,5 @@
 
+// eslint-disable-next-line no-unused-vars
```

Comment:
praise: Good call leaving this comment for future travelers.

Reply:
Haha that was ESLint bugging me to do it :P